### ISSUE TITLE
Timecodes for "ML Zoomcamp 8.5 - Transfer Learning"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#156]
Youtube video: https://www.youtube.com/watch?v=WKHylqfNmq4

**Comment by abhinit21** [comment]
What is this Issue label `timecodes` ? Can I contribute @amitfrancis

**Comment by alexeygrigorev** [comment]
I'll talk about it on the course launch

**Comment by dimzachar** [comment]
0:00:00 - Lesson 5: Transfer Learning with Pre-trained Neural Network
0:01:50 - Transfer learning: reusing convolutional layers for training.
0:03:41 - Pre-process input, read from directory, specify batch size.
0:05:41 - Exploring image dataset classes and generator.
0:07:31 - Multi-class classification with one-hot encoding.
0:09:31 - Creating a base model for training.
0:11:45 - Extracting vector representation from non-trainable model.
0:13:58 - Creating and testing a Keras model.
0:15:47 - Drawing 3D cube, converting to vector.
0:17:45 - Converting 3D/2D to 1D using pooling.
0:19:51 - Building neural network with functional style.
0:21:45 - Pooling and dense layer for predictions.
0:23:27 - Optimizers in Keras: Adam recommended default algorithm.
0:25:17 - Optimizing neural network with loss evaluation.
0:26:58 - Using row scores for stable predictions.
0:28:45 - Training model with fit method and epochs.
0:30:35 - Training model and analyzing accuracy fluctuations.
0:32:49 - Training accuracy reaches almost 1, overfitting observed.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
